muacrypt/account.py

```diff
@@ -327,6 +327,53 @@ def process_incoming(self, msg):
             account=self,
         )
 
+    def process_incoming_gossip(self, dec):
+        """ process gossip from encrypted part of incoming mail message
+        and update states information from any gossip header for the
+        To or Cc recipients of the msg.
+
+        :type dec: DecryptMimeResult
+        :param dec: DecryptMimeResult instance as returned from decrypt_mime
+        :rtype: ProcessIncomingResult
+        """
+        print(dec)
+
+        msg = dec
```

Comment:
why name it dec only to rename it to ``msg``? i think it's fine to just use ``msg`` in the function signature.